common/source/docs/common-gcs-only-operation.rst

```diff
@@ -0,0 +1,53 @@
+.. _common-gcs-only-operation:
+
+========================
+Operation Using Only GCS
+========================
+
+While it is generally recommended that a Radio Control system is employed as primary pilot control means for ArduPilot vehicles, it is not a requirement. It is possible to operate a vehicle using only a Ground Control Station (GCS), with or without joystick input. Setup varies vehicle to vehicle, and is detailed below.
+
+
+Capability
+==========
+
+If a joystick is connected to the GCS, then almost the same control capabilities are available as with a radio control transmitter/receiver. Mode changes can be assigned to joystick buttons. If an RC transmitter is used as the joystick, then the same control precision can be obtained. Use of low resolution , cheaper "game" style joysticks will usually provide a less than enjoyable experience and use of high quality joystick or use of an RC transmitter as a joystick (OpenTX provides this capability) is highly recommended.
+
+It is also possible to use only the GCS itself, if only autonomous missions or guided mode operation is desired.
+
+Failsafes
+=========
+
+It is highly recommended that the GCS failsafe be setup for the vehicle, since GCS connectivity is being relied upon exclusively for vehicle control. See :ref:`apms-failsafe-function` , :ref:`Copter Failsafe <failsafe-landing-page>` , or :ref:`Rover Failsafes <rover-failsafes>`, as appropriate.
+
+Of course, other failsafes can be setup and utilized as desired in addition to the GCS failsafe.
+
+Configuration
+=============
+
+Operation without a RC link requires a few setup changes and these vary between vehicles.
```

Comment:
No big deal but there's a bit of duplication here because half a page up we say, "Setup varies vehicle to vehicle, and is detailed below.".

Reply:
fixed